src/solvers/dg.jl

```diff
@@ -40,6 +40,11 @@ standard textbooks.
   Nodal Discontinuous Galerkin Methods: Algorithms, Analysis, and
   Applications
   [doi: 10.1007/978-0-387-72067-8](https://doi.org/10.1007/978-0-387-72067-8)
+
+`VolumeIntegralWeakForm()` is only implemented for conserved terms as 
+non-conservative terms should always be discretized in conjunction with a flux-splitting scheme, 
+see `VolumeIntegralFluxDifferencing(volume_flux)`.
```

Comment:
```suggestion
see [`VolumeIntegralFluxDifferencing`](@ref).
```
This should insert a link to the other docstring in our online documentation.